config_src/drivers/nuopc_cap/mom_cap.F90

```diff
@@ -650,6 +651,10 @@ subroutine InitializeAdvertise(gcomp, importState, exportState, clock, rc)
   ocean_public%is_ocean_pe = .true.
   call ocean_model_init(ocean_public, ocean_state, time0, time_start, input_restart_file=trim(restartfiles))
 
+  ! GMM, this call is not needed for NCAR. Check with EMC.
+  ! If this can be deleted, perhaps we should also delete ocean_model_flux_init
+  call ocean_model_flux_init(ocean_state)
```

Comment:
@jiandewang  Did you check whether this was needed?

Reply:
@DeniseWorthen no I didn't test that.
there are some remaining issue (CFC related) in this PR, GFDL and NCAR will address them in the next round.